src/views/Scenario/Scenario.js

```diff
@@ -228,6 +245,30 @@ const Scenario = () => {
     label: scenarioListLabel,
     validationStatus: scenarioValidationStatusLabels,
   };
+  // FOR DEVELOPMENT PURPOSE ONLY, DELETE BEFORE MERGE
+  const lastRun = '2023-03-13T16:25:30.795486885Z';
+  // FOR DEVELOPMENT PURPOSE ONLY, DELETE BEFORE MERGE
+
+  const hasScenarioBeenRun = useMemo(
+    () => currentScenario?.data?.lastRun !== null && currentScenario?.data?.state === 'Successful',
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [currentScenario?.data?.lastRun]
```

Comment:
`currentScenario?.data?.state` is missing from the dependency array, was it done on purpose?

Reply:
I decided to use only one dependency because `currentScenario?.data?.state` is closely linked to `currentScenario?.data?.lastRun` (it changes only after a run) so no need to make a double check but if it can be necessary, I will put it in dependency array.

Reply:
Currently, we have no need to try to optimize performance early, we can probably add both in the dependencies array :slightly_smiling_face: 
Furthermore, adding only one of them seems like a risk for bugs when we change the code in the future :smile: